CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/parser/ExcelImportFileParser.java

```diff
@@ -237,23 +237,22 @@ private String getCellStringValue(final Cell cell) {
 
         try {
             switch (type) {
-                case STRING:
+                case STRING -> {
                     result = cell.getStringCellValue();
```

Comment:
you could extract `result` out i.e. `result = switch (type) {...`. You can use `yield` for the `NUMERIC,FORMULA` case